Changed examples and adding a note on the Prometheus scrape_interval … (#12760)
canonical/lxd: doc/metrics.md

      #      is not covered by the certificate (not in the SAN list)
      server_name: 'foo'
```

> 

Reviewer: This empty line is required (that's the reason for the failure of the doc linter).

* The `scrape_interval` is assumed to be 15s by the Grafana Prometheus data source by default.
If you decide to use a different `scrape_interval` value, you must change it in both the Prometheus configuration and the Grafana Prometheus data source configuration.
Otherwise the Grafana `$__rate_interval` value will be calculated incorrectly and possibly cause a `no data` response in queries using it.

* The `server_name` must be specified if the LXD server certificate does not contain the same host name as used in the `targets` list.
  To verify this, open `server.crt` and check the Subject Alternative Name (SAN) section.

  For example, assume that `server.crt` has the following content:

> 

Reviewer: Please also indent the following lines (since they are part of the second bullet point). "Since the ..." should still be indented, the back ticks should not.

      #      is not covered by the certificate (not in the SAN list)
      server_name: 'foo'
```
```{note}
The `scrape_interval` is assumed to be 15s by the Grafana Prometheus data source by default.

> When using a different `scrape_interval` value, change it too in the Grafana Prometheus data source configuration.

Reviewer: ```suggestion
If you decide to use a different `scrape_interval` value, you must change it in both the Prometheus configuration and the Grafana Prometheus data source configuration.
```

Otherwise the Grafana `$__rate_interval` value will be calculated incorrectly and possibly cause a `no data` response in queries using it.
```

> ````{note}

Reviewer: We shouldn't have two consecutive notes - please change to one note with a bullet list for the different points.

Author: > We shouldn't have two consecutive notes - please change to one note with a bullet list for the different points.

Limited experience on formatting, this is the closest I get.

      #      is not covered by the certificate (not in the SAN list)
      server_name: 'foo'
```

> ```{note}

Reviewer: ```suggestion
````{note}
```

Usually 3 backticks are enough, but not in this case since the note include another directive.
So this one is trickier than usual. :wink:

* The `scrape_interval` is assumed to be 15s by the Grafana Prometheus data source by default.

> If you decide to use a different `scrape_interval` value, you must change it in both the Prometheus configuration and the Grafana Prometheus data source configuration.
> Otherwise the Grafana `$__rate_interval` value will be calculated incorrectly and possibly cause a `no data` response in queries using it.

Reviewer: ```suggestion
  If you decide to use a different `scrape_interval` value, you must change it in both the Prometheus configuration and the Grafana Prometheus data source configuration.
  Otherwise the Grafana `$__rate_interval` value will be calculated incorrectly and possibly cause a `no data` response in queries using it.
```

* The `server_name` must be specified if the LXD server certificate does not contain the same host name as used in the `targets` list.

> To verify this, open `server.crt` and check the Subject Alternative Name (SAN) section.
> 
> For example, assume that `server.crt` has the following content:

Reviewer: ```suggestion
  To verify this, open `server.crt` and check the Subject Alternative Name (SAN) section.

  For example, assume that `server.crt` has the following content:
```

Everything that is part of the bullet point needs the same indentation, so please add 2 spaces for the following lines up until and including the "Since the Subject Alternative Name ..." line.
